packages/rollup-plugin/src/rollup-bundle-analysis/rollupBundleAnalysisPlugin.ts

```diff
@@ -25,8 +25,13 @@ export const rollupBundleAnalysisPlugin: BundleAnalysisUploadPlugin = ({
         return;
       }
 
+      let format: typeof options.format | "esm" = options.format;
+      if (format === "es") {
+        format = "esm";
+      }
```

Comment:
```suggestion
      const format = options.format === "es" ? "esm" : options.format;
```

Ditto with usage in vite plugin.

Reply:
Done